openclaw_hooks/nima-recall-live/lazy_recall.py

```
@@ -0,0 +1,468 @@
+            dimensions += 1
+    
+    if dimensions == 0:
+        return 0.5
+    
+    avg_diff = total_diff / dimensions
+    # Convert to similarity (0 diff = 1.0, 1.0 diff = 0.0)
+    resonance = 1.0 - min(avg_diff, 1.0)
+    return resonance
+
+
+def compute_affect_bleed(memories, current_affect, bleed_factor=0.08):
```

Ruff (0.15.0)

[warning] Unused function argument: `current_affect`

(ARG001)